docs/cadence/language/values-and-types.mdx

```diff
@@ -1315,6 +1315,69 @@ are available for both variable-sized and fixed-sized or variable-sized arrays.
     let invalidMapFunctionExample = fixedSizedExample.map(functionAcceptingBool)
     ```
 
+-
+    ```cadence
+    access(all)
+    fun filter(_ f: (T: Bool)): [T]
```

Comment:
```suggestion
    fun filter(_ f: fun(T): Bool): [T]
```

Reply:
while at it, can you please also update the signature for the array `map` function above

Reply:
Done, also updated for `map`.